src/test/locale/tg.js

```diff
@@ -16,14 +17,6 @@ test('parse', function (assert) {
         );
     }
 
-    function equalTestStrict(input, mmm, monthIndex) {
```

Comment:
Why are these tests removed?

Reply:
@orif-jr

Reply:
This code block was newer used for tests. So, it's [blocked the builds always](https://travis-ci.org/github/moment/moment/jobs/727752429#L234). I reverted the code block back and added some more tests.